iModelCore/ECPresentation/Source/Shared/ECExpressions/ECExpressionOptimizer.cpp

```diff
@@ -67,6 +67,9 @@ static bset<ECClassId> GetClassIds(NavNodeKeyCR key)
         ids.insert(key.AsECClassGroupingNodeKey()->GetECClassId());
     else if (key.AsECPropertyGroupingNodeKey())
         ids.insert(key.AsECPropertyGroupingNodeKey()->GetECClassId());
+    else if (key.AsLabelGroupingNodeKey() && key.AsLabelGroupingNodeKey()->GetGroupedInstanceKeys() != nullptr)
+        for (auto const& instanceKey : *key.AsLabelGroupingNodeKey()->GetGroupedInstanceKeys())
+            ids.insert(instanceKey.GetClassId());

```

Comment:
Formatting - if a block has more than 1 line (no matter if it's one or more statements), it should be wrapped with curly braces